Himotoki/decode.swift

```diff
@@ -13,6 +13,11 @@ public func decode<T: Decodable where T.DecodedType == T>(object: AnyObject) thr
 }
 
 /// - Throws: DecodeError
+public func decode<T: Decodable where T.DecodedType == T>(object: AnyObject, rootKeyPath: KeyPath) throws -> T {
```

Comment:
Length Violation: Line should be 100 characters or less: currently 112 characters